middleware/compress/lzw.go

```diff
@@ -0,0 +1,33 @@
+package compress
+
+import (
+	"compress/lzw"
+	"io"
+
+	"goyave.dev/goyave/v5/util/errors"
+)
+
+// Lzw encoder for the gzip format using Go's standard `compress/lzw` package.
+//
+// Takes an Order specifying the bit ordering in an LZW data stream level,
+// and number of bits to use for literal codes, litWidth, as parameters.
+// Accepted values are defined by constants
+// in the standard `compress/lzw` package.
+type Lzw struct {
+	Order    lzw.Order
+	LitWidth int
```

Comment:
Can you please add comments here explaining these two fields and their constraints? You can copy-paste the comments from the `compress/lzw` package. Tell `Order` is optional and its default is `lzw.LSB`.

I'm wondering if we should use a `LitWidth` of 8 as a default if it's not provided, for convenience.